mrp_bom_image/models/mrp_bom.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2024 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class MrpBom(models.Model):
+    _inherit = "mrp.bom"
+
+    image = fields.Binary(related="product_tmpl_id.image_1024")
```

Comment:
naive question : why not use image_1024 ?

Reply:
seems like a mistake ;)
After a brief check, seems like image_1024 is the norm so i will change this !

Reply:
voilà :) :heavy_check_mark: